### DISCUSSION TITLE
Scalar CSS variables create a border inside Docusaurus code blocks

### DISCUSSION THREAD

**Discussion by metsaleht** [scalar/scalar#2177] [General]
Hi, 

Am I doing something wrong? Everything is set up correctly; however, Scalar CSS code seems to create a border inside Docusaurus code blocks. 

--scalar-border-color: var(--ifm-color-emphasis-200, #444950); seems to be the what causes it, resulting in a box shadow. 

<img width="1062" alt="Screenshot 2024-06-19 at 19 58 43" src="https://github.com/scalar/scalar/assets/158271573/b48f9204-a31a-42e2-a054-5250f70d0192">

It should look like this

<img width="1069" alt="Screenshot 2024-06-20 at 10 43 19" src="https://github.com/scalar/scalar/assets/158271573/fb42e189-87fb-4495-aaea-65c440b875c9">

**Comment by hanspagel** [comment]
We refactored a lot that touches this, can you double check that you're on the latest version? 😳
